portfolio/src/main/java/com/google/sps/servlets/LoadCommentsServlet.java

```diff
@@ -34,26 +34,66 @@
 @WebServlet("/load-comments")
 public class LoadCommentsServlet extends HttpServlet {
 
+  int maximumComments = 10;
+
   @Override
   public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
     Query query = new Query("Comment").addSort("timestamp", SortDirection.DESCENDING);
 
     DatastoreService datastore = DatastoreServiceFactory.getDatastoreService();
     PreparedQuery results = datastore.prepare(query);
 
+    int count = 0;
     List<Comment> comments = new ArrayList<>();
+    comments.add(new Comment(0, Integer.toString(results.countEntities()), 0));
```

Comment:
In stead of having a dummy comment object in the list to only store number of comments, why not calculate number of comments in js since you already have a list.

Reply:
I am not sure how I would count it in javascript because the json file that is sent in the response to the javascript file only contains the amount of comments desired by the user, not the total amount of comments. I will look into how to get rid of the dummy!